### PR TITLE
cleanup: change css to minimize the alert box

### LINKED ISSUE #419: Frontend - reduce deposit/payment snackbar height + vertical align middle
image

### PR DESCRIPTION
**Pull request by Shabirmean**
### Background 
- Update the css to minimize the alert box for deposit

### Change Summary
- Override inherited padding to make the box smaller

### Additional Notes
- N/A

### Testing Procedure
- View changes in my cluster: http://35.247.96.212/
- You can either host your own cluster or check in the CI cluster
- Create port forward to the frontend: `kubectl port-forward <POD_ID> -n <NAMESPACE> 8080:8080`
- Access the Pod via `localhost:8080`
- Make a deposit and see if the alert box is compact compared to the one in http://bank-of-anthos.xyz/

### Related PRs or Issues 
- N/A

### Before
image

### After
image

### CODE REVIEW

**Comment by bourgeoisor** [comment]
#419 also alls for vertically aligning the checkmark icon with the text (they're vertically off from each other by a handful of pixels in your screenshots)

**Comment by Shabirmean** [comment]
> #419 also calls for vertically aligning the checkmark icon with the text (they're vertically off from each other by a handful of pixels in your screenshots). The text appears vertically centered, it's the checkmark icon the looks biased towards the bottom of the alert.

I have updated it. Please TAL!

**Comment by ahrarmonsur** [comment]
`8px` padding on the snackbar  doesn't look quite balanced; it is perhaps too narrow. Based on the examples for the [Snackbar specs](https://material.io/components/snackbars#specs) on Material UI, I'd suggest setting the top and bottom padding to be `1em`.

image

Otherwise, LGTM!

**Review by ahrarmonsur — APPROVED** [review]
One suggestion regarding the padding of the snackbar; leaving it to Shabir's discretion. Otherwise, changes LGTM.

**Comment by Shabirmean** [comment]
@ahrarmonsur - Ive updated as per your suggestion.

#### Before
image
---
#### After
image

**Comment by ahrarmonsur** [comment]
Thanks for making the changes, Shabir! Looks good to merge!

File: .github/workflows/ci-pr.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ on:
       - master
 jobs:
   code-tests:
-    runs-on: [self-hosted, live-v2]
+    runs-on: [self-hosted, runners-v2]
```
Comment (Shabirmean) [inline .github/workflows/ci-pr.yaml:22]: Revert this file before merge!

**Pull request merged**